slices/dbus.yaml

```diff
@@ -0,0 +1,35 @@
+package: dbus
+
+essential:
+  - dbus_copyright
+
+slices:
+  bins:
+    essential:
+      - dbus-bin_bins
+      - dbus-daemon_bins
+      - dbus-system-bus-common_config
+      - dbus_config
+      - libc6_libs
+      - libdbus-1-3_libs
+      - libexpat1_libs
+      - libsystemd0_libs
+    contents:
+      /etc/init.d/dbus:
```

Comment:
This should be in `config`.

Reply:
Are we sure? It's a binary

Reply:
Ah, okay. I did not realise this was an executable.